packages/EasyCodingStandard/bin/ecs-container.php

```diff
@@ -2,21 +2,20 @@
 
 use Symfony\Component\Console\Input\ArgvInput;
 use Symplify\EasyCodingStandard\DependencyInjection\ContainerFactory;
-use Symplify\PackageBuilder\Configuration\ConfigFilePathHelper;
-use Symplify\PackageBuilder\Configuration\LevelConfigShortcutFinder;
+use Symplify\PackageBuilder\Configuration\ConfigFileFinder;
+use Symplify\PackageBuilder\Configuration\LevelFileFinder;
 
 require_once __DIR__ . '/easy-coding-standard-bootstrap.php';
 
 // 1. Detect configuration from --level
```

Comment:
Should we change this comment?

Reply:
How?

Reply:
```diff
-Detect configuration from --level
+Detect configuration from the level argument
```

Or something, I just don't like a hardcoded `--level` :)

Reply:
I see :)